src/api/index.js

```diff
@@ -18,14 +18,16 @@ const config = new Conf();
  * Checks if the plugin/package exists on npm
  *
  * @param {String} plugin - The name of the plugin/package
- * @return {Promise} - Throws an error if the plugin/package doesn't exist
+ * @return {Promise} - True if the plugin is found
  */
 const checkOnNpm = plugin => new Promise(resolve => {
-  npmName(plugin).then(available => {
-    if (available) {
-      throw new Error(ERR_MODULE_NOT_FOUND);
+  npmName(plugin).then(notFound => {
```

Comment:
I agree it is really confusing but `npm-name` was created by the original author to check if it's free/available instead. That logic is correct in itself but we're using it for it's inverse value (taken/unavailable).